doc/configuration/Configuration.md

```diff
@@ -0,0 +1,319 @@
+## Mattermost Config Settings
+___
+Config settings allow system administrators to customize their teams Mattermost experience. Warning: system administrators should be cautious to only grant admin privileges to team members who understand the critical nature of the settings they are adjusting.
+
+
+## System Console Settings:
+These settings can be modified from the System Console user interface or config.json.
+
+### Service Settings
+___
+General settings to configure the listening address, login security, testing, webhooks and service integration of Mattermost. 
+
+### System
+
+```"ListenAddress": ":8065"```
+*The IP address to listen and the port which to bind. Entering ":8065" will bind to all interfaces or you can choose one like "127.0.0.1:8065". Changing this will require a server restart before taking effect.*
+
+```"MaximumLoginAttempts": 10```
+*Failed login attempts allowed before user is locked out and required to reset password via email.*
+
+```"SegmentDeveloperKey": ""```
+*For users running SaaS services, sign up for a key at Segment.com to track metrics.*
+
+```"GoogleDeveloperKey": ""```
+*Set this key to enable embedding of YouTube video previews based on hyperlinks appearing in messages or comments. Instructions to obtain a key available at https://www.youtube.com/watch?v=Im69kzhpR3I. Leaving field blank disables the automatic generation of YouTube video previews from links.*
```

Comment:
Should be "Leaving the field blank" instead of "Leaving field blank".

And are we sure about this? I thought the generation of the previews still happened but just didn't pull extra information like uploader name, video title, etc.?